pkg/addonmanager/controllers/agentdeploy/controller.go

```diff
@@ -28,6 +28,7 @@ import (
 	worklister "open-cluster-management.io/api/client/work/listers/work/v1"
 	clusterv1 "open-cluster-management.io/api/cluster/v1"
 	workapiv1 "open-cluster-management.io/api/work/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
why use this? I would not use this for not found error

Reply:
removed